src/components/Modal/Modal.stories.tsx

```diff
@@ -1,83 +1,133 @@
-import { useArgs } from '@storybook/client-api';
+import Button from '@components/Button';
+import Container from '@components-layout/Container';
+import Flexbox from '@components-layout/Flexbox';
+import { css } from '@emotion/react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { MouseEventHandler, useState } from 'react';
 
-import Modal, { Button } from '.';
+import Modal from '.';
 
 export default {
-  title: 'Modal',
+  title: 'Design System/Components/Modal',
   component: Modal,
   parameters: {
     layout: 'fullscreen',
+    componentSubtitle:
+      'Modal은 특정 컨텐츠가 담긴 화면을 다른 화면 위로 띄워줍니다.',
+    docs: {
+      description: {
+        component: `- 다음과 같은 컴포넌트를 children으로 사용할 수 있습니다.  
+          - \\<Modal.Header\\> : \\<Modal\\> Header로 올 내용을 넣어줄 때 사용합니다.
+          - \\<Modal.Content\\> : \\<Modal\\> Content로 올 내용을 넣어줄 때 사용합니다.
+          `,
+      },
+    },
   },
+  argTypes: {
+    isOpen: {
+      name: 'isOpen',
+      description: 'Modal을 띄울지 여부를 나타내는 상태입니다.',
+      defaultValue: true,
+      table: {
+        type: { summary: 'boolean', required: true },
+      },
```

Comment:
```suggestion
table: {
    type: { summary: 'boolean', required: true },
    defaultValue: { summary: true },
      },
```

이렇게 변경해야 table에 정상적으로 뜰 것 같아요

그나저나 default가 true인가요?? 👀

Reply:
헉 아뇨 defautValue 없어요! 지워야겠네요 감사해요~~